display_list/dl_builder.cc

```diff
@@ -15,7 +15,28 @@
 
 namespace flutter {
 
-#define DL_BUILDER_PAGE 4096
+// 1.048576 MB
+static const constexpr uint64_t kDLBuilderMaxGrowth = 1048576;
+
+/// @brief Return the next power of 2.
+///
+/// If the provided value is a power of 2, returns as is.
+uint64_t NextPowerOfTwo(uint64_t x) {
```

Comment:
This goop can be replaced with `std::bit_ceil(x)`? I remember I couldn't use it earlier because it was C++ 17 only.

Reply:
It seems like this is C++ 20 only